qgrid/grid.py

```diff
@@ -20,10 +20,24 @@ def template_contents(filename):
 SLICK_GRID_CSS = template_contents('slickgrid.css.template')
 SLICK_GRID_JS = template_contents('slickgrid.js.template')
 
+SLICK_GRID_DEFAULT_OPTIONS = {
+    'enableCellNavigation': True,
+    'fullWidthRows': True,
+    'syncColumnCellResize': True,
+    'forceFitColumns': True,
+    'rowHeight': 28,
+    'enableColumnReorder': False,
+    'enableTextSelectionOnCells': True, }
+
+
+def show_grid(data_frame, *args, **kwargs):
+    return SlickGrid(data_frame, *args, **kwargs)
+
 
 class SlickGrid(object):
 
-    def __init__(self, data_frame, remote_js=False):
+    def __init__(self, data_frame, remote_js=False, precision=None,
+                 options={}):
```

Comment:
Thanks, while I agree that this is a common gotcha, in this case I had thought about it and since options is only read later (in `self.options.update(options)`) I don't believe this is an issue here. I thought that is more self-documenting than the standard approach of `options=None` and then later:

    if options is None:
        options = {}

I'm happy to change it to the latter though if you are concerned but I don't think it's necessary.